tasks/download-atom-shell-task.coffee

```diff
@@ -68,8 +68,8 @@ module.exports = (grunt) ->
 
         # Make sure atom is executable on Linux
         if process.platform is 'linux'
-          atomAppPath = path.join(directoryPath, 'atom')
-          fs.chmodSync(atomAppPath, '755') if fs.existsSync(atomAppPath)
+          electronAppPath = path.join(directoryPath, 'electron')
```

Comment:
Downloading older versions will still use 'atom'.  In another part of the file [line 138], the name to use is chosen by looking at the filename to see if it's the old version